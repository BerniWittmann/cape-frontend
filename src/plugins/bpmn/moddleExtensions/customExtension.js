export default {
  name: 'CaPE',
  uri: 'http://uni-ulm/schema/bpmn/cape',
  prefix: 'cape',
  xml: {
    tagAlias: 'lowerCase'
  },
  types: [{
    name: 'ExtensionArea',
    superClass: ['bpmn:Task'],
    properties: []
  }],
  emumerations: [],
  associations: []
}
