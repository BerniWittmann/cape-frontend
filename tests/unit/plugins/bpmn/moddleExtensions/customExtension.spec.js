import customExtension from '@/plugins/bpmn/moddleExtensions/customExtension'

describe('Plugins', () => {
  describe('BPMN', () => {
    describe('ModdleExtensions', () => {
      describe('customExtension', () => {
        it('exports an object', () => {
          expect(customExtension).toEqual(expect.any(Object))
        })

        it('has the correct configuration Keys', () => {
          expect(customExtension).toMatchSnapshot()
        })
      })
    })
  })
})
