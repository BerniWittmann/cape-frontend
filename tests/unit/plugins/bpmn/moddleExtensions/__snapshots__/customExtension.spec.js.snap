// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Plugins BPMN ModdleExtensions customExtension has the correct configuration Keys 1`] = `
Object {
  "associations": Array [],
  "emumerations": Array [],
  "name": "CaPE",
  "prefix": "cape",
  "types": Array [
    Object {
      "name": "ExtensionArea",
      "properties": Array [],
      "superClass": Array [
        "bpmn:Task",
      ],
    },
  ],
  "uri": "http://uni-ulm/schema/bpmn/cape",
  "xml": Object {
    "tagAlias": "lowerCase",
  },
}
`;
